### DISCUSSION TITLE
Suggession for a new feature

### DISCUSSION THREAD

**Discussion by warmachine028** [cyblogerz/Hospital-Management-software#9] [Ideas]
<h2>Add this new feature named:</h2>

**DOCTORS AVAILABLE:** 

     1. Add Doctors: 
     Take the following inputs from the user
         i. Doctor id
        ii. Doctor Name
       iii. Specialization
        iv. Designation

     2. Remove Doctors
        i. Ask for doctor id to the USER
       ii. Check if the Doctor id is present in the database
      iii. if present prompt user to confirm deletion and the delete
       iv. if not present print "Invalid Doctor ID"

     3. Update Doctors 
         i. Ask for the Doctor id
        ii. if present:
            a) Ask for details to modify or press '.' to go back.
             NOTE: Doctor Id cannot be modified as it is the primary Key

     4. Display Available Doctors 
          Print all doctor details

**Comment by pranavajy** [comment]
> ## Add this new feature named:
> 
> **DOCTORS AVAILABLE:**
> 
> ```
>  1. Add Doctors: 
>  Take the following inputs from the user
>      i. Doctor id
>     ii. Doctor Name
>    iii. Specialization
>     iv. Designation
> 
>  2. Remove Doctors
>     i. Ask for doctor id to the USER
>    ii. Check if the Doctor id is present in the database
>   iii. if present prompt user to confirm deletion and the delete
>    iv. if not present print "Invalid Doctor ID"
> 
>  3. Update Doctors 
>      i. Ask for the Doctor id
>     ii. if present:
>         a) Ask for details to modify or press '.' to go back.
>          NOTE: Doctor Id cannot be modified as it is the primary Key
> 
>  4. Display Available Doctors 
>       Print all doctor details
> ```

Great Suggestion Bro, Will look into it

**Comment by pranavajy** [comment]
@warmachine028  Bro can you help in resolving some issues
